cms/djangoapps/contentstore/views/course.py

```diff
@@ -1422,6 +1453,18 @@ def group_configurations_detail_handler(request, course_key_string, group_config
             return JsonResponse(status=204)
 
 
+def has_course_enabled_content_experiments(course):
+    """
+    Returns True if Studio should show the "Experiment Group
```

Comment:
The docstring as written describes one possible usage of the method that doesn't. There could be a difference between when content experiments have been enabled, and when Studio should show the section. The reason to rename this is to make it more generic, and the docstring should match that IMO.